kernel/kernel-api/src/main/java/io/delta/kernel/internal/commitrange/CommitRangeImpl.java

```diff
@@ -96,13 +98,32 @@ public List<FileStatus> getDeltaFiles() {
   @Override
   public CloseableIterator<ColumnarBatch> getActions(
       Engine engine, Snapshot startSnapshot, Set<DeltaLogActionUtils.DeltaAction> actionSet) {
+    validateParameters(engine, startSnapshot, actionSet);
+    // Build on top of getCommits() by flattening and adding version/timestamp columns
+    CloseableIterator<CommitActions> commits = getCommits(engine, startSnapshot, actionSet);
+
+    return TableChangesUtils.flattenCommitsAndAddMetadata(engine, commits);
+  }
+
+  @Override
+  public CloseableIterator<CommitActions> getCommits(
```

Comment:
Can you name this `getCommitActions` or something actually? This is misleading, I thought maybe it was getting a list of commit files or something